_sections/home-english.md

```diff
@@ -172,10 +173,33 @@ In general, choice names should be unique within a single-choice list. If two ch
 
 #### Specify other
 
-{% include alerts/warning.html content="We generally recommend using [relevance](#relevant) to specify your own **other** choice. The shortcut described in this section only works for selects without translations or **choice_filter**s. It uses English for the \"Specify other\" choice which cannot be customized." %}
```

Comment:
Why revert this warning? In general, this PR looks like it is an alternative to https://github.com/XLSForm/xlsform.github.io/pull/244 closer to the old content. I prefer the visible warning (https://xlsform.org/en/#specify-other) and not duplicating the relevance section. I think we should only make the change you propose here if there's a very strong reason for it.